app/gui/src/dashboard/components/dashboard/column/LabelsColumn.tsx

```diff
@@ -79,6 +96,36 @@ export default function LabelsColumn(props: column.AssetColumnProps) {
           <ManageLabelsModal backend={backend} item={item} />
         </DialogTrigger>
       )}
+      {isOverflowing && (
+        <Popover.Trigger>
+          <Button
+            variant="icon"
+            showIconOnHover
+            icon={DotsIcon}
+            tooltip={getText('showAllLabels')}
+          />
+          <Popover
+            triggerRef={rootRef}
+            size="auto"
+            style={() => ({ width: rootRef.current?.clientWidth })}
```

Comment:
Popover doesn't have to be the rootRef width.

Alternatively we can use a menu instead

Reply:
yeah not sure. doesn't *have* to be but i think visually it might make the most sense to be the same width. not sure wdym by menu

Reply:
I meant using `<Menu />` component to display labels, but on the second thought I'm not sure it's a good idea